src/Backend/FakeNews/FakeNews.Bll/Roles/UserRoles.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FakeNews.Bll.Roles
+{
+    public static class UserRoles
```

Comment:
I think an enum would be better.

Reply:
The Identity excpects a string value. I chose the static class because with enum you can only assign an int value to the the roles. Using Enum.ToString() would be fine as long as nobody renames any value of this enum. In that case the roles would be completely broken. I still thing this is a more straightforward solution.

Reply:
Okay, I accept this.